tests/cdc-test-decoding/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       -c ssl_cert_file=/etc/ssl/certs/ssl-cert-snakeoil.pem
       -c ssl_key_file=/etc/ssl/private/ssl-cert-snakeoil.key
   target:
-    image: postgres:13-bullseye
+    image: postgres:16-bullseye
```

Comment:
postgres:13-bullseye has issues when dealing with multi-value insert with generated columns.

e.g.

the following fails on PG13

```
insert into generated_column_test(id, name, greet_hello, email) values
(10, 'Tiger', DEFAULT, 'tiger@wild.com'), (2, 'Tiger', DEFAULT, 'tiger@wild.com');
```

but works!

```
insert into generated_column_test(id, name, greet_hello, email) values
(10, 'Tiger', DEFAULT, 'tiger@wild.com');
```

Reply:
Ok, but we still need/want support for Postgres 13 even on the target side of things. Are you saying that the problem is with how the unit-test is written, or is the problem in how we generate the SQL for the target to apply?

Reply:
The problem happens with multi value insert + generated column place holder as DEFAULT.

If we want to support PG 13, either we need to stop using DEFAULT place holder for generated columns(i.e. remove generated column from insert statement) or disable multi value insert optimisation.

Reply:
@dimitri FYI, few tests uses PG16 as target => https://github.com/dimitri/pgcopydb/blob/590e72703dc7e701a5cbd9dd42282e102f7f3a17/tests/follow-wal2json/docker-compose.yml#L19

Reply:
Ideally we would have test coverage for all supported Postgres versions, both as source and target. Having some diversity is better than just using a single Postgres version. Thanks to that we discovered the issue here.

Then sorry about the DEFAULT work you have added, please revert back to actually removing columns and values from the statements.

Reply:
> or is the problem in how we generate the SQL for the target to apply?

The problem is how we generate the multi value INSERT for the target with generated column. It works fine with we exclude columns if it is generated instead of using DEFAULT as a place holder.

Reply:
@dimitri How about disabling multi value INSERT optimization if the target is <PG14?

Reply:
Is it possible to selectively do that only when the target table contains generated columns? if yes, then I think that's a good idea.

Reply:
Other option is to exclude the generated column name and DEFAULT value from INSERT statement based on isGenerated flag. I will explore that.

Reply:
@dimitri I updated the PR to add a new type called LogicalMessageAttribute which holds column name and attributes (currently it has isgenerated). For insert, the new code discards generated column based on isgenerated, for update it is still uses DEFAULT as value.